includes/Feed/FeedDataExporter.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook\Feed;
+
+defined( 'ABSPATH' ) || exit;
+
+use SkyVerge\WooCommerce\Facebook\Feed\FeedProductFormatter;
+
+/**
+ * A class responsible for manipulating output from FeedProductFormatter to technical requirements of the Facebook Feed CSV format.
+ *
+ * @since 2.6.0
+ */
+class FeedDataExporter {
+
+	/**
+	 * Various feed utilities for extracting product information from WC_Product to feed format.
+	 *
+	 * @var FeedProductFormatter Various feed utilities.
+	 */
+	protected $product_formatter;
+
+	/**
+	 * Cached attributes variants.
+	 * Collecting parent information may be costly. There is a big change that a batch of products
+	 * will consist of a group of variations items sharing the same parent. In that case we can cache
+	 * variants attributes and re-used them for each child.
+	 *
+	 * @since 2.6.0
+	 * @var array $attribute_variants Array of variants attributes.
+	 * @see FeedProductFormatter::prepare_product_for_feed()
+	 */
+	protected $attribute_variants = array();
+
+
+	/**
+	 * Return an array of columns to export.
+	 *
+	 * @since  2.6.0
+	 * @return array
+	 */
+	public function get_column_names() {
+		return array(
```

Comment:
I'm curious about the duplication of keys and values in this array? Is there a reason they're the same, will they be different?

Reply:
Not really, this is just a leftover from the requirements of WooCommerce CSV code. We can change it now.